returns/primitives/container.py

```diff
@@ -68,6 +68,18 @@ def __setstate__(self, state: _PickleState | Any) -> None:
             # backward compatibility with 0.19.0 and earlier
             object.__setattr__(self, '_inner_value', state)
 
+    def __replace__(self, /, **changes) -> 'BaseContainer':
```

Comment:
There are no other changes that we allow. Only `inner_value`. Use a single param.